### ISSUE TITLE
Contact variables

### ISSUE THREAD

**Issue by tbeu** [oestersoetebier/IdealizedContact#21]
It would be of interest to have the contact variables like penetration depth, contact normal force, contact tangential force, contact duration etc. as results available.

**Comment by oestersoetebier** [comment]
You are right. It's on the todo list but not yet done completely. The variables are somewhere but do not have consistent names, so one needs to know where to look at.

**Comment by tbeu** [comment]
Do you plan to update the models for the constact variables?

**Comment by oestersoetebier** [comment]
We are planing a new release by the end of this year (or maybe beginning of next year) where we'll try to reduce simulation effort etc. 
This will include well-arranged contact variables. Do you have a specific problem for which you need them? If that's the case, maybe I can help you getting the information you need from the model.

**Comment by tbeu** [comment]
Thanks for letting us know. Glad to hear that development progresses.

Personally I am interested in the contact force over contact time (contact hysteris) as result.

**Comment by tbeu** [comment]
> We are planing a new release by the end of this year (or maybe beginning of next year)

Any news on the roadmap?

**Comment by tbeu** [comment]
> Any news on the roadmap?

Can you please give a hint when an updated version can be expected? Thank you very much.

**Comment by tbeu** [comment]
Any update news after one year silence? Is this project still alive?
